plugins/chat/db/migrate/20221122070108_save_chat_allowed_groups_site_setting.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+class SaveChatAllowedGroupsSiteSetting < ActiveRecord::Migration[7.0]
+  def up
+    chat_enabled = DB.query_single("SELECT value FROM site_settings WHERE name = 'chat_enabled'")
```

Comment:
very minor but maybe we can increase speed of that by where `WHERE value ='t' AND LIMIT 1`

Reply:
LIMIT is not required, no other setting will have the same name. I will add the value thing tho :+1: